plugins/frontend/backstage-plugin-argo-cd/src/components/Test.tsx

```
@@ -0,0 +1,437 @@
+import { styled } from '@mui/system';
+import {
+  DEFAULT_FRAGMENT_SOURCE,
+  GetFragmentResult,
+} from '@roadiehq/catalog-common';
+import { EntityRefLinks, useEntity } from '@backstage/plugin-catalog-react';
+import {
+  Entity,
+  EntityRelation,
+  stringifyEntityRef,
+} from '@backstage/catalog-model';
+import { groupBy } from 'lodash';
```

ESLint

[error] lodash must be declared in dependencies of plugins/frontend/backstage-plugin-argo-cd/package.json, run 'yarn --cwd plugins/frontend/backstage-plugin-argo-cd add lodash' from the project root.

(@backstage/no-undeclared-imports)